src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ContentPresenter/ContentPresenter_NativeEmbedding.xaml.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+using Windows.UI.Core;
+using Uno.UI.Samples.Controls;
+
+namespace Uno.UI.Samples.Content.UITests.ContentPresenter
+{
+	[Sample("ContentPresenter", Name = "ContentPresenter_NativeEmbedding")]
+	public sealed partial class ContentPresenter_NativeEmbedding : UserControl
+	{
+		public ContentPresenter_NativeEmbedding()
+		{
+			this.InitializeComponent();
+		}
+	}
+
+	public sealed partial class NativeControlHostWithText : ContentControl
```

Comment:
We'll need to document how to create nuget packages that can use the airspace integration.

Reply:
Let's do that in a separate PR after testing it thoroughly. We may need to change the API a little bit and/or notice some specific points that need attention in the docs.